validate.lic

```diff
@@ -47,6 +47,11 @@ class DRYamlValidator
       respond("  Validating settings for #{settings.hunting_file_list.size} settings files. Note: yaml errors from #{checkname}-setup.yaml may be duplicated below.")
       respond("")
       settings.hunting_file_list.each do |file|
+        if !File.exist?("./scripts/profiles/#{checkname}-#{file}.yaml")
```

Comment:
```suggestion
       unless File.exist?("./scripts/profiles/#{checkname}-#{file}.yaml")
```

Reply:
Robert prefers the use `if` in lieu of unless. Been trying to make that a habit.

Reply:
In this particular case I'd prefer/recommend function chaining:
```suggestion
    settings.hunting_file_list
    .reject { |file| File.exist?("./scripts/profiles/#{checkname}-#{file}.yaml") }
    .each do |file|
        respond(...)
```